### PR TITLE
Implement new token for gateway in location

### PR DESCRIPTION
**Pull request by cpprian**
## 📖 Description

1. include a **summary of the changes and the related issue**, eg. _Closes #XYZ_
2. Do not make a PR if you can't check **all the boxes below**

#### Documentation ###

- [ ] If testing requires changes in the environment or deployment, please **update the documentation** (https://defguard.gitbook.io) first and **attach the link to the documentation** section in this pool request
- [ ] I have commented on my code, particularly in hard-to-understand areas

#### Testing ### 

- [ ] I have prepared end-to-end tests for all new functionalities
- [ ] I have performed end-to-end tests manually and they work
- [ ] New and existing unit tests pass locally with my changes

#### Deployment ###

- [ ] If deployment is affected I have made corresponding/required changes to [deployment](https://github.com/defguard/deployment) (Docker, Kubernetes, one-line install)

#### Testing ### 

- [ ] I have merged my changes before to dev and the dev checklist is done
- [ ] I have tested all functionalities on the dev instance and they work

#### Documentation ###

- [ ] I have made corresponding changes to the **user & admin documentation** and added new features documentation with screenshots for users/admins

### CODE REVIEW

File: web/src/pages/network/NetworkGateway/NetworkGateway.tsx [hunk 1]
```diff
@@ -50,15 +59,63 @@ export const NetworkGatewaySetup = () => {
...
+            title={LL.gatewaySetup.card.authToken()}
+            disableExpand={true}
+            expanded={true}
+            actions={getNetworkTokenActions}
+          >
+            <p>{returnNetworkToken()}</p>
+          </ExpandableCard>
+        </>
+      )}
+      {/* Docker Based Gateway Setup */}
```
Comment (filipslezaklab) [inline web/src/pages/network/NetworkGateway/NetworkGateway.tsx:113]: unnecessary

File: web/src/pages/network/NetworkGateway/NetworkGateway.tsx [hunk 2]
```diff
@@ -50,15 +59,63 @@ export const NetworkGatewaySetup = () => {
...
-      </header>
+      <section className="header-section">
+        <h2>{LL.gatewaySetup.header.main()}</h2>
+        {parse(
+          LL.gatewaySetup.messages.runCommand({
+            setupGatewayDocs: externalLink.gitbook.setup.gateway,
+          }),
+        )}
+      </section>
+      {/* Authentication Token */}
```
Comment (filipslezaklab) [inline web/src/pages/network/NetworkGateway/NetworkGateway.tsx:91]: unnecessary

File: web/src/pages/network/NetworkGateway/NetworkGateway.tsx [hunk 3]
```diff
@@ -76,6 +133,25 @@ export const NetworkGatewaySetup = () => {
...
+      {/* From Package */}
+      <h3>{LL.gatewaySetup.header.fromPackage()}</h3>
+      <MessageBox>
+        {parse(
+          LL.gatewaySetup.messages.fromPackage({
+            setupGatewayDocs: externalLink.gitbook.setup.gateway,
+          }),
+        )}
+      </MessageBox>
+      <Button
```
Comment (filipslezaklab) [inline web/src/pages/network/NetworkGateway/NetworkGateway.tsx:145]: Just wrap this button in `<a>` tag with target of "_self"

File: web/src/pages/network/NetworkGateway/NetworkGateway.tsx [hunk 4]
```diff
@@ -50,15 +59,63 @@ export const NetworkGatewaySetup = () => {
...
+        onClick={() => {
+          writeToClipboard(returnNetworkToken());
+        }}
+      />,
+    ],
+    [returnNetworkToken, writeToClipboard],
+  );
+
+  // TODO: consider a better way to redirect to the gateway releases page
+  const handleSubmit = () => {
```
Comment (filipslezaklab) [inline web/src/pages/network/NetworkGateway/NetworkGateway.tsx:77]: See comment on button

File: web/src/pages/network/NetworkGateway/style.scss [hunk 5]
```diff
@@ -24,10 +31,43 @@
           @include small-text;
 
           line-height: 22px;
+          color: var(--text-body-secondary);
         }
       }
 
       margin-bottom: 3rem;
```
Comment (filipslezaklab) [inline web/src/pages/network/NetworkGateway/style.scss:38]: space blocks with padding if possible

**Review by filipslezaklab — CHANGES_REQUESTED** [review]

File: web/src/pages/network/NetworkGateway/NetworkGateway.tsx (hunk 3 above)
Comment (cpprian) [reply web/src/pages/network/NetworkGateway/NetworkGateway.tsx:145]: For now unnecessary, maybe in the future this could be available

**Review by filipslezaklab — APPROVED** [review]

**Pull request merged**